### PR TITLE
Add a step to create a js folder

### PR DESCRIPTION
**Pull request by scottaddie**
Fixes https://github.com/aspnet/AspNetCore.Docs/issues/14620

### CODE REVIEW

File: aspnetcore/tutorials/web-api-javascript.md [hunk 1]
```diff
@@ -40,11 +40,13 @@ The simplest `fetch` call accepts a single parameter representing the route. A s
 
 1. Create a *wwwroot* directory in the project root.
 
+1. Create a *js* directory inside of the *wwwroot* directory.
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/web-api-javascript.md]: Create  a *wwwroot/js*  folder to contain the JavaScript assets.

Comment (Rick-Anderson) [reply aspnetcore/tutorials/web-api-javascript.md]: I'd consider global replace directory with folder.

File: aspnetcore/tutorials/web-api-javascript.md [hunk 2]
```diff
@@ -40,11 +40,13 @@ The simplest `fetch` call accepts a single parameter representing the route. A s
...
+
 1. Add an HTML file named *index.html* to the *wwwroot* directory. Replace its contents with the following markup:
 
     [!code-html[](first-web-api/samples/3.0/TodoApi/wwwroot/index.html)]
 
-1. Add a JavaScript file named *site.js* to the *wwwroot* directory. Replace its contents with the following code:
+1. Add a JavaScript file named *site.js* to the *wwwroot/js* directory. Replace its contents with the following code:
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/web-api-javascript.md]: 1. Add a JavaScript file named *site.js* to the *wwwroot/js* folder. Replace the contents of the *site.js* file with the following code:

File: aspnetcore/tutorials/web-api-javascript.md (hunk 1 above)
Comment (guardrex) [reply aspnetcore/tutorials/web-api-javascript.md]: Discussed at https://github.com/aspnet/AspNetCore.Docs/pull/12708#discussion_r290829685.

Comment (Rick-Anderson) [reply aspnetcore/tutorials/web-api-javascript.md]: I meant in this doc

Comment (guardrex) [reply aspnetcore/tutorials/web-api-javascript.md]: Yeah ... just mentioning that in passing. His reasoning is there, and I selected some quotes from guidance. If you want a firm rule followed, let me know. I would need to sweep the topics that I wrote. I used "folder" most of the time.

Comment (Rick-Anderson) [reply aspnetcore/tutorials/web-api-javascript.md]: No need to sweep. I prefer folder. Let's standardise new work on folder.

Comment (Rick-Anderson) [reply aspnetcore/tutorials/web-api-javascript.md]: Sometimes directory sounds better. In this case, folder is shorter and sounds better.

**Pull request merged**